### ISSUE TITLE
Update to newer LLVM version

### ISSUE THREAD

**Issue by TheDan64** [TheDan64/limonite#59]
Blocked by Inkwell lacking support for versions other than 3.7 at the moment. Will try and update as many newer versions as we can.

**Comment by TheDan64** [comment]
See https://github.com/TheDan64/inkwell/issues/1 for Inkwell tracking issue

**Comment by TheDan64** [comment]
This is no longer blocked on inkwell as it supports 3.6 - 5.0 (6.0 soon)

**Comment by TheDan64** [comment]
This is done in `features/new` which will be merged into master soon
